cms/envs/common.py

```diff
@@ -1103,3 +1099,14 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+########## django-sudo ##########
+if FEATURES.get('ENABLE_DJANGO_SUDO', False):
```

Comment:
This isn't going to work, if the FEATURE flag is set in `aws.py`. (Because this happens before flags get modified in that file)